ntbs-service/Models/Validations/ValidRiskFactorAttribuite.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using ntbs_service.Models.Enums;
+
+namespace ntbs_service.Models.Validations
+{
+    public class ValidRiskFactorAttribute : ValidationAttribute
+    {
+        protected override ValidationResult IsValid(object value, ValidationContext validationContext)
+        {
+            var riskFactor = (RiskFactorBase) value;
+
+            var isAnyChosen = riskFactor.InPastFiveYears || riskFactor.IsCurrent || riskFactor.MoreThanFiveYearsAgo;
+            if (riskFactor.Status == Status.Yes && !isAnyChosen) {
+                return new ValidationResult(GetErrorMessage());
+            } else {
+                return ValidationResult.Success;
+            }
+        }
+
+        private string GetErrorMessage()
+        {
+            return "At least one field should be selected";

```

Comment:
Can you put this string in the `ValidationHelper`? It'll be good to have them all in one place so they can be reviewed together